graph/graph.go

```diff
@@ -22,15 +22,16 @@ const (
 )
 
 type Node struct {
-	ID        string                 // unique identifier for the node
-	NodeType  string                 // Node type
-	Namespace string                 // Namespace
-	Workload  string                 // Workload (deployment) name
-	App       string                 // Workload app label value
-	Version   string                 // Workload version label value
-	Service   string                 // Service name
-	Edges     []*Edge                // child nodes
-	Metadata  map[string]interface{} // app-specific data
+	ID         string                 // unique identifier for the node
+	NodeType   string                 // Node type
+	Namespace  string                 // Namespace
+	Workload   string                 // Workload (deployment) name
+	App        string                 // Workload app label value
+	Version    string                 // Workload version label value
+	Service    string                 // Service name
+	Edges      []*Edge                // child nodes
+	Metadata   map[string]interface{} // app-specific data
+	Accessible bool                   // If the node is accessible or not
```

Comment:
In graph/cytoscape/cytoscape.go you use the "Inaccessible" semantic and here the positive one.
It sounds to me better to use same semantic in both places.